tests/property/ic_api/cycles/test/test.ts

```diff
@@ -0,0 +1,13 @@
+import { getCanisterId } from 'azle/dfx';
+import { runTests } from 'azle/test';
+
+import { createActor } from './dfx_generated/intermediary';
+import { getTests } from './tests';
+
+const intermediaryCanister = createActor(getCanisterId('intermediary'), {
```

Comment:
Always use that one function thing called something like getCanisterActor

Reply:
See if the top-level await will work here

Reply:
it doesn't because of jest